app/views/oauth_applications/_form.html.erb

```diff
@@ -0,0 +1,63 @@
+<%= form_for application, url: doorkeeper_submit_path(application), html: { class: 'form-horizontal', role: 'form'} do |f| %>
+  <% if application.errors.any? %>
+    <div class="alert alert-warning alert-dismissible fade show" role="alert">
+      <h5>
+        <%= t('doorkeeper.applications.form.error') %>
```

Comment:
hier in der ganzen datei (und allen anderen *.html.erb) auch gern die rails teile double-quoten `"..."`